modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -426,11 +426,11 @@ Replaced by: xref:reference/procedures.adoc#procedure_dbms_cluster_secondaryrepl
 | label:yes[]
 | label:admin-only[] label:deprecated[Deprecated in 5.9]
 
-// New in 5.16 but only on Aura
+// New in 5.16 on Aura, from Neo4j 5.17 also on-prem
```

Comment:
It seems like the JAR has (unintentionally or not) actually been shipped for a few versions already... but arguably 5.17 is still the official release, I suppose?

Reply:
😄

Reply:
It's anyway just a comment.